.pre-commit-config.yaml

```diff
@@ -2,7 +2,7 @@ default_language_version:
   python: python3
 repos:
 - repo: https://github.com/psf/black
-  rev: 24.10.0
```

Comment:
I saw this opposite in your other PR, which one is correct? https://github.com/RedHatInsights/insights-rbac/pull/1205/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9L12